config.yml

```diff
@@ -5,13 +5,21 @@ defaults:
   # TaskCluster credentials
   taskcluster:
     rootUrl: !env TASKCLUSTER_ROOT_URL
+    queue: none
     credentials:
       clientId: !env TASKCLUSTER_CLIENT_ID
       accessToken: !env TASKCLUSTER_ACCESS_TOKEN
 
   monitoring:
     enable: !env:bool MONITORING_ENABLE
 
+  worker:
+    workerId: "succeed"
+    workerType: "succeed"
+    workerGroup: "garbage-hybrid1999"
+    provisionerId: "garbage-hybrid1999"
```

Comment:
Let's set both of these (workerGroup and provisionerId) to `built-in` in the production configuration.